000-template.md

```diff
@@ -1,29 +0,0 @@
-<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
```

Comment:
I don't think you wanted to delete this here, or?

Reply:
Good spot - I did not! Must have messed up a rebase. Will restore it in a new commit